code/ATMOSPHERICS/pipe/construction.dm

```diff
@@ -202,11 +205,18 @@ var/list/bent_dirs = list(NORTH|SOUTH, WEST|EAST)
 	else
 		src.pipe_type = pipe_type
 		src.dir = dir
+	if(src.pipe_type == PIPE_BSCAP)
+		src.w_class = W_CLASS_LARGE
+		bspipe_item_list.Add(src)
```

Comment:
Probably unnecessary since you already add it in New()

Reply:
there are 2 lists: bspipe_list and bspipe_item_list, one tracks the caps in item form and the other in machinery form